posts/why-is-react-native-so-promising.md

```diff
@@ -0,0 +1,161 @@
+
+React Native still being in development, the dev team is experimenting with some other engines. They recently updated the JSCore version, resulting in a 40% increase in performance.
+
+## Styling and Animations
+
+Although not being CSS, the styling in React Native is heavily inspired by it. Design is made using the StyleSheet module and consists merely of declaring a JS object with CSS inspired properties. Not every CSS feature is supported. For example, no nesting is allowed, and animations are supported through a separate Animation API mechanism.
+
+Concerning the components positioning, you'll only use the flex properties (it's like CSS flex work), which is great when you want your application to work across a variety of device resolutions.
+
+```
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    borderRadius: 4,
+    borderWidth: 0.5,
+    borderColor: '#d6d7da',
+  },
+  title: {
+    fontSize: 19,
+    fontWeight: 'bold',
+  },
+  activeTitle: {
+    color: 'red',
+  },
+});
+```
+
+A big advantage of defining your style in JavaScript is dynamic styling: you don't have to have a style defined only during code time or compile time. With React Native you can compose your styles and properties, combine them and then have them applied onto your component. 
+
+This dynamic styling is basically how the animations are made. React Native provides two APIs: Animated and LayoutAnimation; the first one is used to make specific interactions whereas the latter, as its name suggest, is more layout oriented. It will offer less control than Animated but won't require as much calculation to render the animation. Note that this is an experimental feature on Android.
+
+## Or you can use styled-components 💅
+
+Along with the HOC and the stateless components, [styled-components](https://github.com/styled-components/styled-components) are my 2nd revelation of the year. This growing React library allows you to create atomic CSS styled components by mixing HTML components and CSS style. 
+
+First of all, no, it is not a CSS in JS kind of library. It spits out CSS classes (with obfuscated class names). The main feature of this library is that it uses the power of ES6's string templates; you can use the component's props and state directly in its style and let style-component do its magic.
+
+Because the style-components dev team is awesome,~~they made it compatible with React Native with only one simple change~~ Because they're **really awesome** they improved the use for React Native in their 3rd major release by making it seamless. A single import will be compatible with React and React Native.
```

Comment:
Missing "." before the second "Because"?

Reply:
lower cased the B